src/pages/wallets/find-wallet.tsx

```diff
@@ -237,14 +237,11 @@ const FindWalletPage = ({ data, location }) => {
       <Flex
         px={{ base: 0, md: 8 }}
         pt={4}
-        pb={0}
+        pb={6}
         w="full"
         gap={6}
-        height="90vh"
-        overflow="hidden"
-        position="sticky"
         top="76px"
-        mb={{ base: "230px", md: "120px", lg: "150px" }}
+        mb={{ base: "30px", md: "20px", lg: "50px" }}
```

Comment:
Wondering why do we need these hardcoded values for this spacing.

Could we just do something like
```suggestion
        mb={16}
```

would result in
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/7f16029f-70e6-47eb-a815-6be21228d2f3)